### ISSUE TITLE
change node versions in table

### ISSUE THREAD

**Issue by yairper** [compat-table/node-compat-table#22]
Hi guys,
1. All of node 4.x versions uses v8 4.5.103
   same goes for 5.x which uses v8 4.6.85
   IMO, there is no need to separate them into different columns
2. in node 6.x it uses v8 5.0.71
   until 6.5 which bumped to v8 5.1.281
   IMO, they should have separate columns

WDYT?

**Comment by williamkapke** [comment]
Here's the how I chose the versions:
1) always a nightly
2) show the 3 latest of v6, v5, & v4 (v6 appears to not be working correctly!)
4) show latest of v0.12 & v0.10 because some folks still use them and it was requested

This is done here:
https://github.com/williamkapke/node-compat-table/blob/gh-pages/versions.js#L30

It makes complete sense to go by the v8 version. But... it IS possible for Node core to cause tests to fail/pass. I haven't noticed that happen though.

The bottom line is: It should show whatever the visitors want to see (even if the v8 version is the real differentiator). The thing is, I don't know what the visitors expect to see! I can only guess- so your feedback is great. PR welcome!

**Comment by pbyairp** [comment]
will do

**Comment by yairper** [comment]
Will do

**Comment by yairper** [comment]
It seems like that's a major code change,
the "versions" flow will become redundant if I'm not mistaken,
are you ready for such a change?

**Comment by williamkapke** [comment]
Hm. I'm probably not...? I was thinking it was a simple update to `versions.js`.

It is mostly just important that the latest versions are used as soon as they are released. I don't really know a good way to do versioning based on v8 version- so I was hoping you had some simple changes you could drop in a PR, but if it is a ton of work- then maybe not.

Can you elaborate on how you would go about it?

**Comment by yairper** [comment]
I think that I would go for a static .versions file (instead of generating it)
and put there the versions which contains different v8
0.10, 0.12, 4, 5, 6, 6.5
WDYT?

**Comment by williamkapke** [comment]
I think long term, I would like to see the versions as `7.x`, `6.x`, `5.x`, `4.x`, `12.x`, `10.x` and hovering over them shows a dialog that enumerates all version that were tested in that column. If there are ANY differences in minor or patch version, then they would get their own column. For example we might see `>4.5.x` and `<4.5.x` because something cause a difference.

This will take a significant amount of work but I think it is the best solution.

**Comment by yairper** [comment]
I will pass
